text/0000-issue-workflow.md

```diff
@@ -0,0 +1,222 @@
+- type/chore
+
+# How it Works
+[how-it-works]: #how-it-works
+
+This is the technical portion of the RFC, where you explain the design in sufficient detail.
+
+### Standard Workflow
+
+```text
+     +----------+  +----------------------+            +----------+  +----------------------+
+     |             |                      |            |             |                      |
+     |             |     status/triage    |            |             |    status/triage     |
+     |             |                      |         P  |             |                      |
+     |             +----------------------+         U  |             +----------------------+
+     |                         |                    L  |                         |
+  I  |                         |                    L  |                         |
+  S  |             +-----------v----------+            |             +-----------v----------+
+  S  |             |                      |         R  |             |                      |
+  U  |             |     status/ready     |         E  |             | status/code-approved |
+  E  |             |                      |         Q  |             |                      |
+  S  |             +----------------------+         U  |             +----------------------+
+     |                         |                    E  |                         |
+     |                         |                    S  |                         |
+     |             +-----------v----------+         T  |             +-----------v----------+
+     |             |                      |         S  |             |                      |
+     |             |  status/in-progress  |            |             | status/user-accepted |
+     |             |                      |            |             |                      |
+     +----------+  +----------------------+            +----------+  +----------------------+
+                               |                                                 |
+                               |                                                 |
+                               |                                                 |
+                               v                                                 v
+                             CLOSED                                            CLOSED
+```
+
+
+### Actionable Issues
+
+An actionable issue is one that can be worked on such that the following is true:
+
+- Definition of done - that may be acceptance criteria or clear expectations.
+- Appropriate labels
+    - `type/*`
+    - `size/*` (if possible)
+    - `status/ready`
+    - `good first issue` (if applicable)
+
+
+### Triage
+
+The following is a flow assist contributors in assigning the appropriate labels in script format.
+
+#### Issues
+
+- Is this a support request?
+    - Yes
+        - Add label type/support
+        - Provide support and close the issue with label `status/resolved`\
```

Comment:
I think we should prod the author of the issue to close it when they feel the question has been addressed and only close ourselves after a period of inactivity.